src/pages/RepoPage/BundlesTab/BundleContent/BundleContent.tsx

```diff
@@ -24,13 +26,17 @@ const Loader = () => (
 
 const BundleContent: React.FC = () => {
   const { provider, owner, repo } = useParams<URLParams>()
+  const { newBundleTab } = useFlags({
```

Comment:
Just curious, looking at the hook and are the function params / response obj documented anywhere? Not blocking for this PR

Reply:
Uh, they're injected via the LaunchDarkly React integration, there's no real way atm for us to document them without going into LD itself.

Reply:
Got it, I'll poke around